Source/Renderer/AutomaticUniforms.js

```diff
@@ -1478,6 +1478,26 @@ define([
             }
         }),
 
+         /**
+         * An automatic GLSL uniform representing the splitter position to use when rendering imagery layers with a splitter.
+         * This will be in the range 0.0 to 1.0 with 0.0 being the far left of the viewport and 1.0 being the far right of the viewport.
+         *
+         * @alias czm_imagerySplitPosition
+         * @glslUniform
+         *
+         *
+         * @example
+         * // GLSL declaration
+         * uniform float czm_imagerySplitPosition;
+         */
+        czm_imagerySplitPosition : new AutomaticUniform({
```

Comment:
Can you also add a unit test for this?  It is easy, see [Specs/Renderer/AutomaticUniformSpec.js](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Specs/Renderer/AutomaticUniformSpec.js).